invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/InvitationActionButtons.js

```diff
@@ -9,30 +9,49 @@
 import React from "react";
 import { Button } from "semantic-ui-react";
 import { i18next } from "@translations/invenio_communities/i18next";
+import _isEmpty from "lodash/isEmpty";
+import PropTypes from "prop-types";
 
-export const ViewButton = ({ onClick }) => (
-  <Button size="tiny" content={i18next.t("View")} onClick={onClick} />
-);
+const ViewButton = ({ onClick, show }) =>
```

Comment:
could you move the buttons below without separating a new component for them ? I think it is a bit overkill